src/createTRPCJotai.ts

```diff
@@ -15,7 +15,7 @@ import type { inferObservableValue } from '@trpc/server/observable';
 
 import { atom } from 'jotai/vanilla';
 import type { Atom, Getter, WritableAtom } from 'jotai/vanilla';
-import { atomWithObservable } from 'jotai/vanilla/utils';
+import { atomWithObservable, atomWithRefresh } from 'jotai/vanilla/utils';
```

Comment:
> "jotai": ">=1.11.0"

`atomWithRefresh` isn't available in older versions. Maybe, we should implement it without it.

Reply:
Okay, I've implemented the refresh feature manually like you suggested